website/docs/index.html.markdown

```diff
@@ -56,14 +56,17 @@ If you are using this provider on your local command line without remote operati
 [remote state backend](https://www.terraform.io/docs/state/remote.html)), there
 are two more options available to you:
 
-- **Use `terraform login` to generate credentials:** Using this provider with
-Terraform on your local command line, you can use the credentials generated by
-[`terraform login`](https://www.terraform.io/docs/commands/login.html) by
-setting the value of the environment variable `TERRAFORM_CONFIG` to the location
-of the file created (for example `~/.terraform.d/credentials.tfrc.json` on Linux
-systems).
-- **Setting a `credentials` block in your CLI config file (.terraformrc):** See
+- **Use `terraform login` to generate credentials:** When using this provider with
+Terraform on your local command line, it can automatically discover the credentials generated by
+[`terraform login`](https://www.terraform.io/docs/commands/login.html).
+- **Set a `credentials` block in your CLI config file (.terraformrc):** See
 the [CLI Configuration File documentation](/docs/commands/cli-config.html).
+    - If you used the `TF_CLI_CONFIG_FILE` environment variable to specify a
+      non-default location for .terraformrc, the provider will also use that
+      location.
+    - If you want the provider to use credentials from a different config file
+      than Terraform itself uses, set the `TERRAFORM_CONFIG` environment
+      variable to the file path the provider should use.
```

Comment:
```suggestion
If you used the `TF_CLI_CONFIG_FILE` environment variable to specify a
non-default location for .terraformrc, the provider will also use that location.
```

Bah, I forgot to re-update the docs after re-discovering the history of those two environment variables. We should return `TERRAFORM_CONFIG` to its resting place in obscurity!